scheduler/src/cook/rest/api.clj

```diff
@@ -28,6 +28,7 @@
             [compojure.core :refer [ANY GET POST routes]]
             [cook.compute-cluster :as cc]
             [cook.config :as config]
+            [cook.rest.cors :as cors]
```

Comment:
Was this intentional?

Reply:
nope, don't even know how it got there haha. deleted!